pkg/apis/pipeline/v1beta1/pipeline_validation.go

```diff
@@ -270,6 +297,16 @@ func validatePipelineWorkspaces(wss []WorkspacePipelineDeclaration, pts []Pipeli
 			}
 		}
 	}
+	for tIdx, t := range finalTasks {
+		for wsIdx, ws := range t.Workspaces {
```

Comment:
this is SO MINOR but in go i think we tend to prefer shorter variable names, so something like `i` and `j` in this case imo would be easier to read in go than `tIdx` and `wsIdx`

Reply:
Done, yup I hear you and with you, was following what we had for `PipelineTask` 😆 , updated `PipelineTask` code as well 👍